src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int INDEX_OF_SUPPLY_LENGTH = 7;
+    private static final int INDEX_OF_BUY_LENGTH = 4;
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        File fileForRead = new File(fromFileName);
+        StringBuilder firstBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileForRead))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                firstBuilder.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't reaad this file", e);
+        }
+        int countSupply = 0;
+        int countBuy = 0;
+        String [] amounts = firstBuilder.toString().split(" ");
+        for (String amount : amounts) {
+            if (amount.contains("supply")) {
+                countSupply += Integer.parseInt(amount.substring(INDEX_OF_SUPPLY_LENGTH,
+                        amount.length()));
+            }
+            if (amount.contains("buy")) {
+                countBuy += Integer.parseInt(amount.substring(INDEX_OF_BUY_LENGTH,
+                        amount.length()));
+            }
```

Comment:
Try to use split method, it will make your logic simpler

Reply:
the split method was used earlier in the code